Build/Projects/MGCB.definition

```diff
@@ -10,6 +10,8 @@
     <Reference Include="System.Xml" />
     <Reference Include="MonoGame.Framework" />
     <Reference Include="MonoGame.Framework.Content.Pipeline" />
+
+    <Reference Include="MGCB.References" />
```

Comment:
> Instead you want to leave the standard platforms (Linux, Windows, etc.) and
> tell them to enable the relevant services instead.

The goal here is to remove the Mac, WindowsGL, and Linux platforms and have one DesktopGL platform.  This means one set of projects and one set of final binaries that work on all three platforms.

What we don't want is to have multiple projects and multiple binaries that all have 100% the same code.

This seems like a reasonable path that should be allowed when the platforms are so similar.  In some ways Android and OUYA could be done like this in the future too.